test/Microsoft.Sbom.Api.Tests/Workflows/ManifestGenerationWorkflowTests.cs

```diff
@@ -401,7 +401,9 @@ await externalDocumentReferenceChannel.Writer.WriteAsync(new ExternalDocumentRef
             Assert.AreEqual(1, noAssertionElements.Count);
 
             var organizationElements = elements.Where(element => element["type"].ToString() == "Organization").ToList();
-            Assert.AreEqual(3, organizationElements.Count);
+            Assert.AreEqual(2, organizationElements.Count);
+            var organizationElementWithSupplierName = organizationElements.Where(element => element["name"].ToString() == "Organization: testSupplier").ToList();
+            var organizationElementWithNoAssertion = organizationElements.Where(element => element["name"].ToString() == "NOASSERTION").ToList();
```

Comment:
These variables don't seem to be used... not sure if you meant to remove them or possibly Assert their contents in some way?

Reply:
They should be asserted, that was an oversight on my part.